doc/nest_by_example/NEST by Example - An Introduction to the Neural Simulation Tool NEST Version 2.12.0.ipynb

```diff
@@ -0,0 +1,2437 @@
+    "  </tr>\n",
+    "  <tr>\n",
+    "    <td class=\"tg-031e\">IE<br></td>\n",
+    "    <td class=\"tg-031e\">E<br></td>\n",
+    "    <td class=\"tg-yw4l\">I<br></td>\n",
+    "    <td class=\"tg-yw4l\">Random convergent $C_{\\text{E}}\\rightarrow 1$, weight $J$, delay $D$</td>\n",
+    "  </tr>\n",
+    "  <tr>\n",
+    "    <td class=\"tg-yw4l\">EI</td>\n",
+    "    <td class=\"tg-yw4l\">I</td>\n",
+    "    <td class=\"tg-yw4l\">E</td>\n",
+    "    <td class=\"tg-yw4l\">Random convergent $C_{\\text{I}}\\rightarrow 1$, weight $-gJ$, delay $D$</td>\n",
+    "  </tr>\n",
+    "  <tr>\n",
+    "    <td class=\"tg-yw4l\">II</td>\n",
+    "    <td class=\"tg-yw4l\">I</td>\n",
+    "    <td class=\"tg-yw4l\">I</td>\n",
+    "    <td class=\"tg-yw4l\">Random convergent $C_{\\text{I}}\\rightarrow 1$, weight $-gJ$, delay $D$</td>\n",
+    "  </tr>\n",
+    "  <tr>\n",
+    "    <td class=\"tg-yw4l\">Ext</td>\n",
+    "    <td class=\"tg-yw4l\">E$_{\\text{ext}}$</td>\n",
+    "    <td class=\"tg-yw4l\">E $\\cup$ I</td>\n",
+    "    <td class=\"tg-yw4l\">Divergent $1 \\rightarrow N_{\\text{E}} + N_{\\text{I}}$, weight $J$, delay $D$</td>\n",
+    "  </tr>\n",
+    "</table>\n",
+    "\n",
+    "<style type=\"text/css\">\n",
+    ".tg  {border-collapse:collapse;border-spacing:0;}\n",
+    ".tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}\n",
+    ".tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}\n",
+    ".tg .tg-e3zv{font-weight:bold}\n",
+    ".tg .tg-hgcj{font-weight:bold;text-align:center}\n",
+    ".tg .tg-yw4l{vertical-align:top}\n",
+    "</style>\n",
+    "<table class=\"tg\" width=90%>\n",
+    "  <tr>\n",
+    "    <th class=\"tg\" style=\"font-weight:bold;background-color:#000000;color:#ffffff;\" colspan=\"2\">D: Neuron and Synapse Model<br></th>\n",
+    "  </tr>\n",
+    "  <tr>\n",
+    "    <td class=\"tg-031e\">**Name**</td>\n",
+    "    <td class=\"tg-031e\">Iaf neuron<br></td>\n",
+    "  </tr>\n",
+    "  <tr>\n",
+    "    <td class=\"tg-031e\">**Type**<br></td>\n",
+    "    <td class=\"tg-031e\">Leaky integrate- and-fire, $\\delta$-current input</td>\n",
+    "  </tr>\n",
+    "  <tr>\n",
+    "    <td class=\"tg-031e\">**Sub-threshold dynamics**<br></td>\n",
+    "    <td class=\"tg-031e\">\\begin{equation*}\n",
+    "      \\begin{array}{rll}\n",
+    "      \\tau_m \\dot{V}_m(t) = & -V_m(t) + R_m I(t) &\\text{if not refractory}\\; (t > t^*+\\tau_{\\text{rp}}) \\\\[1ex]\n",
+    "      V_m(t) = & V_{\\text{r}} & \\text{while refractory}\\; (t^*<t\\leq t^*+\\tau_{\\text{rp}}) \\\\[2ex]\n",
+    "      I(t) = & {\\frac{\\tau_m}{R_m} \\sum_{\\tilde{t}} w\n",
+    "        \\delta(t-(\\tilde{t}+D))}\n",
+    "      \\end{array}\n",
+    "    \\end{equation*}<br></td>\n",
+    "  </tr>\n",
+    "  <tr>\n",
+  
```

Comment:
Line 1995. 
I realize my comment isn't the best one, sorry about that. I think you should switch 'above' and 'code' from the original sentence, and remove 'we look at', so that the sentence becomes 'The first function from the code above is also the first one that is called for every class object.'. But this is just a suggestion :)